api/src/main/java/org/vivoweb/linkeddatafragments/servlet/VitroLinkedDataFragmentServlet.java

```diff
@@ -73,6 +79,16 @@ private File getConfigFile(ServletConfig config) throws IOException {
     public void init(ServletConfig servletConfig) throws ServletException {
         try {
             ServletContext ctx = servletConfig.getServletContext();
+            configProps = ConfigurationProperties.getBean(ctx);
+            
+            if (!configurationPresent()) {
+            	throw new ServletException("TPF not in config");
+            } else {
+            	if (!tpfActiveFlag.equals("true")) {
+            		throw new ServletException("TPF not set to true");
```

Comment:
It may also be helpful to give a hint in this exception text as to where/how the administrator can configure the TPF flag.
```
TPF is currently disabled. To enable, set 'tpfActive.flag=true' in runtime.properties.
```

Reply:
updated now